src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -30,6 +30,10 @@
                                          )
 
 ACRES_PER_SQM = 0.000247105
+SQM_PER_HECTARE = 10000
```

Comment:
Agreed. I think it's more conventional to multiply by a conversion factor instead of dividing.